app_dart/lib/src/service/github_service.dart

```diff
@@ -153,6 +153,11 @@ class GithubService {
     );
   }
 
+  /// Gets label list for an issue.
+  Future<List<IssueLabel>>? getIssueLabels(RepositorySlug slug, int issueNumber) async {
+    return github.issues.listLabelsByIssue(slug, issueNumber).toList();
```

Comment:
Does github.issues.listLabelsByIssue return a nullable list?

Reply:
Good catch! If there are no labels, it returns an empty list. Updated.